public/index.php

```diff
@@ -33,9 +45,9 @@
 }
 
 // Retrieve configuration
-$appConfig = require __DIR__ . '/../config/application.config.php';
-if (file_exists(__DIR__ . '/../config/development.config.php')) {
-    $appConfig = ArrayUtils::merge($appConfig, require __DIR__ . '/../config/development.config.php');
+$appConfig = require realpath($path['app_config']);
+if (file_exists($path['dev_config'])) {
```

Comment:
The code written this way seems inefficient. Before, we just checked the paths directly, while now we put them in an array, then use specific keys of that array? :thinking: